contents/shells/bash.md

```diff
@@ -165,7 +157,9 @@ DESCRIPTION
 
 ```
 ## How to Get Started with linux command-line based shell?
-In my opinion, there is no need to learning linux command-line based shell in particular, because there are tons of linux commands out there, it is impossible to remember and master them all at one time without actual practices. Instead, when it comes to the time that you have to use it, try to use the terminal instead of GUI based tool. Although it might be hard at first, it will benefit you more later. For example, `git`, I highly recommend get started to use command-line based git instead of visualisation tools such as `Source Tree`.
+In our opinion, there is no need to learning linux command-line based shell in particular, because there are tons of linux commands out there, it is impossible to remember and master them all at one time without actual practices. Instead, when it comes to the time that you have to use it, try to use the terminal instead of GUI based tool. Although it might be hard at first, it will benefit you more later. For example, `git`, We highly recommend get started to use command-line based git instead of visualisation tools such as `Source Tree`.
+
+However, if you really wish to learn the bash systematically, below we have listed some resources you might find useful.
```

Comment:
`wish to learn the bash` -> `wish to learn bash`

`below we have listed some resources`... not sure about this. `below are some resources`?